start-all.sh

```diff
@@ -1,6 +1,32 @@
 #!/usr/bin/env bash
 
-sudo docker run -d -v $PWD/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:Z -p 9090:9090 --name aprom prom/prometheus:v1.0.0
+usage="$(basename "$0") [-h] [-d Prometheus data-dir] -- starts Grafana and Prometheus Docker instances"
+
+while getopts ':hd:' option; do
+  case "$option" in
+    h) echo "$usage"
+       exit
+       ;;
+    d) DATA_DIR=$OPTARG
+       ;;
+    :) printf "missing argument for -%s\n" "$OPTARG" >&2
+       echo "$usage" >&2
+       exit 1
+       ;;
+   \?) printf "illegal option: -%s\n" "$OPTARG" >&2
+       echo "$usage" >&2
+       exit 1
+       ;;
+  esac
+done
+
+if [ -z $DATA_DIR ]
+then
+    sudo docker run -d -v $PWD/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:Z -p 9090:9090 --name aprom prom/prometheus:v1.0.0
+else
+    echo "Loading prometheus data from $DATA_DIR"
+    sudo docker run -d -v $DATA_DIR:/prometheus -v $PWD/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:Z -p 9090:9090 --name aprom prom/prometheus:v1.0.0
```

Comment:
I needed to add flag `Z` to `-v $DATA_DIR:/prometheus:Z`, otherwise Prometheus complained that:
```
level=error msg="Error opening memory series storage: open /prometheus/VERSION: permission denied" source="main.go:143" 
```